src/locale/localed-util.c

```diff
@@ -699,6 +697,38 @@ int vconsole_convert_to_x11(const VCContext *vc, X11Context *ret) {
 
                 return x11_context_copy(ret, &xc);
         }
+
+        /* No custom mapping has been found, see if it's the keymap is a converted one as in such case deduce
+         * the conversion to the corresponding x11 layout is easy. */
+        _cleanup_free_ char *xlayout = NULL, *converted = NULL;
+        char *xvariant = NULL;
+
+        xlayout = strdup(vc->keymap);
+        if (!xlayout)
+                return -ENOMEM;
+        xvariant = strchr(xlayout, '-');
+        if (xvariant)
+                *xvariant++ = '\0';
+
+        xc = (X11Context) {
+                .layout  = xlayout,
+                .model   = (char *) "microsoftpro",
```

Comment:
hmm? where does this come from? needs a comment

Reply:
From the commit message:

> Note: by default the automatically generated x11 keyboard configs use keyboard
> model "microsoftpro" which should be equivalent to "pc105" model but with the
> internet/media key mapping added.

However if someone could confirm it that would be nice.

Reply:
maybe add a comment about that to the sources, not just the commit msg